src/components/navbar.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Link, withPrefix } from "gatsby";
+
+import SocialBar from "./socialbar";
+
+const NavBar = (location) => {
+
+  const rootPath = withPrefix("/");
+  let header;
+
+  let logoStyle = {
+    height: "2.5em",
+    marginBottom: 0
+  };
+
+  if (location.pathname === rootPath) {
+    header = (
+      <img className="logo" alt="AdoptOpenJDK logo" src={withPrefix("adopt_logo_white.svg")} style={logoStyle} />
+    );
+  } else {
+    header = (
+      <Link to={"/"} style={{ boxShadow: "none", color: "inherit", lineHeight: "0px" }}>
+        <img className="logo" alt="AdoptOpenJDK logo" src={withPrefix("adopt_logo_white.svg")} style={logoStyle} />
+      </Link>
+    );
+  }
+
+  return (
+    <header id="site-header-main" style={{ display: "flex", backgroundColor: "#152935", background: "url(https://adoptopenjdk.net/dist/assets/background.png)", justifyContent: "center", alignItems: "center", height: "6em"}}>
```

Comment:
Suggestion, takes the local file (reduces risk of being impacted by changes made to the website):

```diff
diff --git a/src/components/navbar.js b/src/components/navbar.js
index 5341c7a..b23f35c 100644
--- a/src/components/navbar.js
+++ b/src/components/navbar.js
@@ -1,6 +1,7 @@
 import React from "react";
 import { Link, withPrefix } from "gatsby";
 
+import Background from "../images/background.png";
 import SocialBar from "./socialbar";
 
 const NavBar = (location) => {
@@ -26,7 +27,7 @@ const NavBar = (location) => {
   }
 
   return (
-    <header id="site-header-main" style={{ display: "flex", backgroundColor: "#152935", background: "url(https://adoptopenjdk.net/dist/assets/background.png)", justifyContent: "center", alignItems: "center", height: "6em"}}>
+    <header id="site-header-main" style={{ display: "flex", backgroundColor: "#152935", backgroundImage: `url(${Background})`, justifyContent: "center", alignItems: "center", height: "6em"}}>
       {header}
       <SocialBar />
     </header>
```

Reply:
changed